examples/java/src/main/java/io/scanbot/sdk/snippets/datacapture/VINScannerSnippet.java

```diff
@@ -0,0 +1,38 @@
+package io.scanbot.sdk.snippets.datacapture;
+
+import io.scanbot.sdk.vin.*;
+import io.scanbot.sdk.ScanbotSDK;
+import io.scanbot.sdk.image.ImageRef;
+import io.scanbot.sdk.licensing.LicenseStatus;
+import io.scanbot.sdk.textpattern.TextPatternScannerResult;
+
+public class VINScannerSnippet {
+    public static void run(ImageRef image) throws Exception {
+        // Make sure you have a valid license
+        if(ScanbotSDK.getLicenseInfo().getStatus() != LicenseStatus.OKAY)
+            return;
+
+        VinScannerConfiguration config = new VinScannerConfiguration();
+        config.setExtractVINFromBarcode(true);
+        // Configure other parameters as needed.
+
+        try(
+            VinScanner scanner = new VinScanner(config)
+        ) {
+            VinScannerResult result = scanner.run(image);
```

Comment:
Can VinScannerResult contain images? If so should be moved to the try-with-resources